--- exodus/CMakeLists.txt	2014-10-15 14:20:05.000000000 -0600
+++ exodus/CMakeLists.txt	2015-03-10 16:09:13.507244468 -0600
@@ -53,6 +53,14 @@
   /usr/local/hdf5/lib
   )
 
+find_library( SZIP_LIBRARY szip
+  $ENV{ACCESS}/lib/shared
+  $ENV{ACCESS}/lib
+  $ENV{NETCDF_DIR}/lib
+  $ENV{NETCDF_DIR}/libsrc/.libs
+  /usr/local/hdf5/lib
+  )
+
 # Check for ZLib, but only if using HDF5.
 FIND_PACKAGE(ZLIB)
 IF(NOT ZLIB_LIBRARY)

--- exodus/forbind/CMakeLists.txt	2014-10-15 14:20:05.000000000 -0600
+++ exodus/forbind/CMakeLists.txt	2015-03-10 16:12:36.946906065 -0600
@@ -40,6 +40,15 @@
   /usr/local/hdf5/lib
   )
 
+find_library( SZIP_LIBRARY szip
+  $ENV{ACCESS}/lib/shared
+  $ENV{ACCESS}/lib
+  $ENV{NETCDF_DIR}/lib
+  $ENV{NETCDF_DIR}/libsrc/.libs
+  /usr/local/hdf5/lib
+  )
+
+
 INCLUDE_DIRECTORIES(
   "${CMAKE_CURRENT_SOURCE_DIR}/include"
   "${CMAKE_CURRENT_SOURCE_DIR}/../cbind/include"
@@ -53,7 +62,7 @@
 
 ADD_LIBRARY(exoIIv2for ${EXODUS_LIBRARY_TYPE} ${forbind_SRCS})
 
-TARGET_LINK_LIBRARIES(exoIIv2for exodus ${NETCDF_LIBRARY} ${HDF5_LIBRARY} ${HDF5HL_LIBRARY} ${ZLIB_LIBRARY} ${CURL} m dl)
+TARGET_LINK_LIBRARIES(exoIIv2for exodus ${NETCDF_LIBRARY} ${HDF5HL_LIBRARY} ${HDF5_LIBRARY} ${ZLIB_LIBRARY} ${SZIP_LIBRARY} ${CURL} m dl)
 
 IF(EXODUS_LIBRARY_PROPERTIES)
   SET_TARGET_PROPERTIES(exoIIv2for PROPERTIES ${EXODUS_LIBRARY_PROPERTIES})

--- exodus/cbind/CMakeLists.txt	2014-10-15 14:20:05.000000000 -0600
+++ exodus/cbind/CMakeLists.txt	2015-03-10 16:10:20.227808782 -0600
@@ -281,7 +281,7 @@
 
 ADD_LIBRARY(exodus ${EXODUS_LIBRARY_TYPE} ${cbind_SRCS})
 
-TARGET_LINK_LIBRARIES(exodus ${NETCDF_LIBRARY} ${HDF5_LIBRARY} ${HDF5HL_LIBRARY} ${ZLIB_LIBRARY} ${CURL} m dl)
+TARGET_LINK_LIBRARIES(exodus ${NETCDF_LIBRARY} ${HDF5HL_LIBRARY} ${HDF5_LIBRARY} ${ZLIB_LIBRARY} ${SZIP_LIBRARY} ${CURL} m dl)
 
 IF(MSVC)
   set_target_properties(exodus PROPERTIES COMPILE_DEFINITIONS _CRT_SECURE_NO_WARNINGS)
