diff -crB --new-file tecio/CMakeLists.txt tecio.patched/CMakeLists.txt
*** CMakeLists.txt	1969-12-31 17:00:00.000000000 -0700
--- CMakeLists.txt	2016-03-15 12:12:59.523998352 -0600
***************
*** 0 ****
--- 1,25 ----
+ cmake_minimum_required(VERSION 2.8)
+ 
+ file(GLOB tecio_src
+     "tecsrc/*.cpp"
+ )
+ 
+ add_definitions(-DUSEENUM -DTHREED -DMAKEARCHIVE)
+ 
+ if (APPLE)
+     add_definitions(-DDARWIN)
+     include_directories( "/usr/X11R6/include" )
+     if (CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+         add_definitions(-DMAC64)
+     endif ()
+ else ()
+     add_definitions(-DLINUX)
+     if (CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+         add_definitions(-DLINUX64)
+     endif ()
+ endif ()
+ 
+ add_library(tecio ${tecio_src})
+ 
+ install(TARGETS tecio DESTINATION lib)
+ install(FILES tecsrc/TECIO.h tecsrc/TECXXX.h DESTINATION include)
